Documentation/ColumnsConfig/Type/Input/Properties/LinkPopup.rst

```diff
@@ -7,29 +7,7 @@
 linkPopup
```

Comment:
This can be removed and redirected to the equivalent `columns-link-properties-appearance`